server/src/main/java/io/crate/analyze/relations/select/SelectAnalyzer.java

```diff
@@ -59,12 +61,14 @@ private static class InnerVisitor extends DefaultTraversalVisitor<Void, SelectAn
 
         @Override
         protected Void visitSingleColumn(SingleColumn node, SelectAnalysis context) {
-            Symbol symbol = context.toSymbol(node.getExpression());
+            Expression expression = node.getExpression();
+            Symbol symbol = context.toSymbol(expression);
             String alias = node.getAlias();
-            if (alias != null) {
-                context.add(ColumnIdent.of(alias), new AliasSymbol(alias, symbol));
+            if (alias == null) {
+                String name = expression instanceof FunctionCall fn ? fn.getName().getSuffix() : null;
```

Comment:
Also worth checking how postgres behaves for fully qualified functions or udfs. To see if they only output the suffix or fully qualified name

Reply:
```
postgres=# select * from pg_catalog.current_database();
 current_database 
------------------
 postgres
(1 row)

postgres=# create schema my_schema;
CREATE SCHEMA
postgres=# CREATE FUNCTION my_schema.inc(val integer) RETURNS integer AS $$
BEGIN
RETURN val + 1;
END; $$
LANGUAGE PLPGSQL;
CREATE FUNCTION

postgres=# select my_schema.inc(1);
 inc 
-----
   2
(1 row)
 
```